src/meta/processors/job/JobManager.cpp

```diff
@@ -396,6 +397,9 @@ nebula::cpp2::ErrorCode JobManager::jobFinished(
       }
       // job has been stopped successfully, so update the job status
       return save(jobKey, jobVal);
+    } else if (code == nebula::cpp2::ErrorCode::E_JOB_NOT_STOPPABLE) {
+      // rollback the job status
+      optJobDesc.setStatus(rollbackStatus, true);
```

Comment:
f***k there's risk stop block task report!

Reply:
This `optJobDesc` is a temporary variable. Where to save it?

Reply:
my bad.

Reply:
fixed by https://github.com/vesoft-inc/nebula/pull/4742